blackbox/docs/sql/functions_operators.txt

```diff
@@ -0,0 +1,21 @@
+.. highlight:: psql
+.. _sql_functions_operators:
+
+=======================
+Functions and Operators
+=======================
+
+This chapter provides an overview of functions and operators that are permitted
+for writing expressions in CrateDB. Operators are reserved keywords that are
+primarily used in a :ref:`sql_dql_where_clause` to perform arithmetic operations or

```

Comment:
you still mention where..... operators are used everywhere, select list, order by, group by, having, join conditions.....